examples/tv-app/linux/include/audio-output/AudioOutputManager.cpp

```diff
@@ -22,36 +22,75 @@ using namespace std;
 using namespace chip::app;
 using namespace chip::app::Clusters::AudioOutput;
 
+AudioOutputManager::AudioOutputManager()
+{
+    mCurrentOutput = 1;
+
+    for (int i = 1; i < 4; ++i)
+    {
+        OutputInfoType outputInfo;
+        outputInfo.outputType = chip::app::Clusters::AudioOutput::OutputTypeEnum::kHdmi;
+        outputInfo.name       = chip::CharSpan::fromCharString("HDMI");
```

Comment:
Maybe add a comment that this is only safe because we are using a string literal?

Reply:
Added.